python/torch_mlir_e2e_test/test_suite/basic.py

```diff
@@ -3098,3 +3099,22 @@ def forward(self):
 @register_test_case(module_factory=lambda: SortIntListReverse())
 def SortIntListReverse_basic(module, tu: TestUtils):
     module.forward()
+
+
+# ==============================================================================
+
+
+def AutomaticallyStripOverloads():
```

Comment:
An e2e test is not appropriate for this kind of test, because we don't actually care about the computation being performed. This would be appropriate as a new file make_fx.py in https://github.com/llvm/torch-mlir/blob/main/python/test/compile_api/

Reply:
Thanks, wasn't quite sure where to put that.